ci-prepush.sh

```
@@ -1,94 +1,253 @@
+    failed_tasks+=("${GRADLE_TASK_ARGS[@]}")
+
+    if [ "$AUTO_FIX_MODE" = "1" ]; then
+        echo ""
+        echo "→ --auto-fix mode: re-running with per-task healers"
+        for entry in "${GRADLE_TASKS[@]}"; do
+            task="${entry%%|*}"
+            heal="${entry#*|}"
+            if [ -n "$heal" ]; then
+                echo ""
+                echo "→ healing $task via $heal"
+                if ./gradlew $heal --build-cache --configuration-cache; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)